backend/clubs/views.py

```diff
@@ -2618,6 +2625,10 @@ def create_tickets(self, request, *args, **kwargs):
                             order_limit:
                                 type: int
                                 required: false
+                            drop_time:
+                                type: string
+                                format: date-time
```

Comment:
nit: can you update the response schema to reflect 403

Reply:
Thanks for catching this, @rm03. Your commitment to detail is bot-like. Appreciate your kind review.